langserver/references.go

```diff
@@ -54,6 +54,10 @@ func (h *LangHandler) handleTextDocumentReferences(ctx context.Context, conn JSO
 	}
 
 	if obj.Pkg() == nil {
+		if _, builtin := obj.(*types.Builtin); builtin {
+			// TODO: support builtin with textDocument/references
```

Comment:
Do you think we'll ever support builtins like this? Seems like it'd take a ton of computation.